aptos-move/framework/aptos-framework/sources/function_info.spec.move

```diff
@@ -1,13 +1,67 @@
 spec aptos_framework::function_info {
     spec module {
-        pragma verify = false;
+        fun spec_is_identifier(s: vector<u8>): bool;
+    }
+
+    // native function
+    spec check_dispatch_type_compatibility_impl(lhs: &FunctionInfo, r: &FunctionInfo): bool {
+        // TODO: temporary mockup
+        pragma opaque;
+    }
+
+    // native function
+    spec load_function_impl(f: &FunctionInfo){
+        // TODO: temporary mockup
+        pragma opaque;
+    }
+
+    // native function
+    spec is_identifier(s: &vector<u8>): bool {
```

Comment:
do we need this spec since we already have a native uninterpreted version of it?

Reply:
This opaque spec connects the function `is_identifier` with `spec_is_identifier`. We could do the same with the native uninterpreted function.